src/modals/SettingsModal/CustomThemeForm.js

```diff
@@ -615,6 +636,11 @@ export default function CustomThemeForm() {
           <Button size="sm" onClick={resetCurrentTheme}>
             Reset values
           </Button>
+          <Button
+            title="Save a copy of your customized theme, you may send it to us to include as a main theme. Importing a theme in the browser is not yet supported."
+            size="sm ml-2" onClick={downloadJsonFile} href={"#"} style={{ 'opacity': '40%'}}>
```

Comment:
```suggestion
            size="sm ml-2" onClick={downloadJsonFile} style={{ 'opacity': '40%'}}>
```
not being a link href is not required